app/views/placements/organisations/index.html.erb

```diff
@@ -2,24 +2,14 @@
   <div class="govuk-grid-row">
```

Comment:
This file doesn't set the `<%= content_for :page_title, t(".organisations") %>`

Reply:
You are right. After all my banging on about this, and then I forgot it!

Reply:
Done